doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc

```diff
@@ -7,13 +7,32 @@ A user task can be assigned to a particular user, multiple users, or to a group.
 
 image::admin-and-config/task-inbox.png[Task inbox]
 
-Users with the `process-admin` role can view and manage all user tasks from the *Tasks* page in {CENTRAL}, located under *Menu* -> *Manage* -> *Tasks*. Users with the `admin` role can access the *Tasks* page but do not have access rights to view and manage tasks by default.
+Process administrators can view and manage all user tasks from the *Tasks* page in {CENTRAL}, located under *Menu* -> *Manage* -> *Tasks*. Users with the `admin` role can access the *Tasks* page but do not have access rights to view and manage tasks by default.
+
+The tasks which are displayed on the *Menu* → *Manage* → *Tasks* page or *Menu* → *Track* → *Task Inbox* page are determined by the current user permissions.
+
+To manage all the tasks, user must be specified as a process administrator by defining any of the following conditions:
+
+* User is specified as `task admin user`. The default value is `Administrator`.
+* User must belong to the task administrators group. The default value is `Administrators`.
+
+Both the values can be configured via `org.jbpm.ht.admin.user` and `org.jbpm.ht.admin.group` system properties.
+
+To change the default value, open `standalone-full.xml` from the {PRODUCT} installation directory, set the following property under the <system-properties> tag:
```

Comment:
"... value of these user system properties, open the `standalone-full.xml` file in the {PRODUCT} installation directory, and set the following property under the `<system-properties>` tag:"

Note several corrections in the above rewrite, either for clarity, consistency, or accuracy.